.github/workflows/integration-tests.yml

```
@@ -11,6 +11,10 @@ on:
 
 jobs:
     test:
+        defaults:
+          run:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 12 but found 10

(indentation)